tests/unit/transaction-details.spec.js

```diff
@@ -13,6 +16,23 @@ Object.assign(Vue.prototype, {
 
 Vue.use(Vuex);
 
+const useGetter = ({ testTransaction }) => (name) => {
+  const getters = {
+    getTx: () => testTransaction,
+    getTxSymbol: () => AETERNITY_SYMBOL,
+    getTxTipUrl: () => '',
+    getTxAmountTotal: () => 1,
+    getExplorerPath: () => 'https://explorer.testnet.aeternity.io/transactions/th_fxSJErbUC3WAqiURFSWhafRdxJC6wzbj5yUKmLTUte6bNWLB8',
+    isTxAex9: () => true,
+  };
+
+  return ['getTx', 'getTxTipUrl', 'getExplorerPath'].includes(name)
+    ? { value: getters[name] }
+    : getters[name];
+};
+
+jest.mock('../../src/composables/vuex.ts');
```

Comment:
In my opinion mocking up functions and the whole structure is a bad thing. This adds too much complexity to the codebase.